client/app/bundles/course/group/pages/GroupShow/GroupTableCard.jsx

```diff
@@ -98,12 +158,24 @@ const GroupTableCard = ({ group }) => {
           <FormattedMessage {...translations.noMembers} />
         </div>
       ) : null}
+      <FormControlLabel
```

Comment:
This should be placed before the table. Let's say we have a very long group table, then we need to scroll to the bottom of the table to find this checkbox

Reply:
okay